### ISSUE TITLE
 Shortened sleep cycle

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5856]
**Preferred term label:**
 Shortened sleep cycle

**Synonyms**
Short sleep

**Definition (free text, please give PubMed ID)**
A tendency to sleep less hours than usual while feeling well rested
PMID: 31473062

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002360

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:618591 
https://pubmed.ncbi.nlm.nih.gov/31473062/  (Frequency in this family is 6/7 carriers)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
